src/previewFrame/elementHighlighterScript.js

```diff
@@ -0,0 +1,36 @@
+const elementHighlighterScript = `(${function() {
+  window.addEventListener('message', (message) => {
+    const data = JSON.parse(message.data);
+    const {selector: {selector: selectorFromMessage}} = data;
+    const selector = selectorFromMessage;
```

Comment:
You can just do `const {selector: {selector}} = data`.

(Or just `const {selector} = data` if you want to remove the extra nesting per my preceding comment)

Reply:
Updated